art/attacks/evasion/adversarial_patch/adversarial_patch.py

```diff
@@ -169,6 +168,19 @@ def reset_patch(self, initial_patch_value: Optional[Union[float, np.ndarray]]) -
         """
         self._attack.reset_patch(initial_patch_value=initial_patch_value)
 
+    def insert_transformed_patch(self, x: np.ndarray, patch: np.ndarray, image_coords: np.ndarray):
+        """
+        Insert patch to image based on given or selected coordinates.
+
+        :param x: The images to insert the patch.
```

Comment:
I think this function is only applied for a single image, so this description confuses me that the patch can be applied for a batch of images.